italia/env/ld/.htaccess

```
@@ -0,0 +1,7 @@
+Header set Access-Control-Allow-Origin * 
+Options +FollowSymLinks 
+RewriteEngine on 
+SetEnvIf Request_URI ^.*$ ROOT_URL=https://dati.isprambiente.it/ld/
+
+RewriteRule ^(.*)$ %{ENV:ROOT_URL}$1 [R=303,L]
+RewriteRule ^(.*)/$ %{ENV:ROOT_URL}$1 [R=303,L]
```

Anything with a trailing slash will also match the first pattern and be replaced in the output.  If you want a trailing optional slash but to exclude it from the output, use this:
```suggestion
RewriteRule ^(.*)/?$ %{ENV:ROOT_URL}$1 [R=303,L]
```
If you want to output the slash if present, you can remove the second rule.